source/module_esolver/esolver_ks_pw_tool.cpp

```diff
@@ -55,7 +55,8 @@ void ESolver_KS_PW<T, Device>::KG(const double fwhmin,
     ModuleBase::GlobalFunc::ZEROS(ct22, nt);
 
     hamilt::Velocity velop(this->pw_wfc, this->kv.isk.data(), &GlobalC::ppcell, &GlobalC::ucell, INPUT.cond_nonlocal);
-    double decut = (wcut + 5*fwhmin)  / ModuleBase::Ry_to_eV;
+    double decut = (wcut + fwhmin)  / ModuleBase::Ry_to_eV;
```

Comment:
Does deleting this `5*` meet expectations?

Reply:
Yes